<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticleEmitterOptionWidget</class>
 <widget class="QWidget" name="ParticleEmitterOptionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="_objectName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QSlider" name="_particuleDeltaAngleEdit">
       <property name="maximum">
        <number>360000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Vert :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Largeur emission :</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Taille :</string>
       </property>
      </widget>
     </item>
     <item row="22" column="1">
      <widget class="QSlider" name="_endColorBlueEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QSlider" name="_particuleSizeEdit">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Rouge :</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="21" column="0">
      <widget class="QLabel" name="_endColorGreenLbl">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Vert :</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QPushButton" name="_chooseEndColorButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Couleur de fin</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QSlider" name="_colorRedEdit">
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="26" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Y :</string>
       </property>
      </widget>
     </item>
     <item row="25" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>X :</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QPushButton" name="_chooseColorButton">
       <property name="text">
        <string>Couleur</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Mise a jour transparance :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Quantit√© :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="_emitterQtyEdit">
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="26" column="1">
      <widget class="QLineEdit" name="_particuleGravityYEdit">
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Mise a jour taille :</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QSlider" name="_colorBlueEdit">
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="_emitterWidthEdit">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSlider" name="_emitterHeightEdit">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Vitesse :</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Delta angle :</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Bleu :</string>
       </property>
      </widget>
     </item>
     <item row="23" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="25" column="1">
      <widget class="QLineEdit" name="_particuleGravityXEdit">
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QCheckBox" name="_updateAlphaEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QCheckBox" name="_updateSizeEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="27" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="10" column="1">
      <widget class="QSlider" name="_particuleAngleEdit">
       <property name="maximum">
        <number>360000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Angle : </string>
       </property>
      </widget>
     </item>
     <item row="21" column="1">
      <widget class="QSlider" name="_endColorGreenEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="20" column="0">
      <widget class="QLabel" name="_endColorRedLbl">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Rouge :</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSlider" name="_particuleLifeEdit">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QSlider" name="_colorGreenEdit">
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QCheckBox" name="_endColorEdit">
       <property name="text">
        <string>Couleur de fin de vie</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Hauteur emission :</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QSlider" name="_particuleSpeedEdit">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Life :</string>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="QSlider" name="_endColorRedEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="22" column="0">
      <widget class="QLabel" name="_endColorBlueLbl">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Bleu :</string>
       </property>
      </widget>
     </item>
     <item row="24" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Gravit√© :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="_particleUseTextureCheckbox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Utiliser texture : </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
